node/src/Transport.ts

```diff
@@ -714,17 +714,20 @@ export class Transport
 			throw new TypeError('if given, appData must be an object');
 		}
 
+		// Clone given RTP parameters to not modify inout data.
```

Comment:
'inout'? This is repeated along the PR

Reply:
Because I just wrote it once (wrongly) and pasted everywhere ☺️

Will fix